count_intersection/README.md

```diff
@@ -37,22 +39,32 @@ python create_intersection_dataset.py \
   - `dataset_folder`: Directory to save the dataset.
   - `dataset_split`: Dataset split (`train` or `validation`).
   - `push_to_hub`: Whether to upload the dataset to the Hugging Face Hub.
+  - `dataset_id`: ID (username + dataset name) used to upload to the Hub. Required if `push_to_hub` is True.
```

Comment:
Would be a nice idea to show an example of `ID`, feel free to use `ariG23498/intersection-dataset` as the example.